src/utils/wsRedirect.ts

```diff
@@ -46,6 +47,7 @@ export class WsRedirect {
 
     // We got a new web socket connection, initiate a TCP connection to the target Intel AMT host/port.
     logger.debug(`${messages.REDIRECT_OPENING_WEB_SOCKET} to ${params.host}: ${params.port}.`)
+    MqttProvider.publishEvent('success', ['handleConnection'], messages.REDIRECTION_SESSION_STARTED)
```

Comment:
Can this call fail? Is any error handling needed?

Reply:
If a MQTT broker is not configured/missing, the call will silently fail without impacting the rest of the code